exe/Main.hs

```diff
@@ -1,9 +1,12 @@
 {-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE ScopedTypeVariables #-}
 
 module Main where
 
 import Control.Monad ( forM )
+import qualified Colog.Core as L
 import Data.Version (showVersion)
+import Data.Text.Prettyprint.Doc
```

Comment:
GitHub Actions scream a bunch of deprecation warnings here. Can we do something about that? I remember, there was an issue with my PR https://github.com/haskell/haskell-language-server/pull/2352

Reply:
Yeah, I'm not sure what the best approach is. Pepe really wants to keep compatibility with old prettyprinter versions. That gives us two options: 1) use the deprecated import, 2) use CPP. I hate 1 less but I'm happy to take the other bad option also.

Reply:
Eh, for now, just leave it be. Eventually we can just drop it, probably.